main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+// /hello endpoint
+func homeLink(c *gin.Context) {
+	c.JSON(200, gin.H{
+		"message": "Hello, World!",
+	})
+}
+
+// /greet/{name} endpoint
+func nameLink(c *gin.Context) {
+	c.JSON(200, gin.H{
+		"message": "Hello, " + c.Param("name"),

```

Comment:
Did you see my previous comment? `"name"` should be a global const, that way the string is not hardcoded.